### ISSUE TITLE
Support for new Places API endpoints

### ISSUE THREAD

**Issue by Tylerlhess** [googlemaps/google-maps-services-python#509]
I would like to see support for the new Places API endpoints.

**Comment by wangela** [comment]
If you would like to upvote the priority of this issue, please comment below or react on the original post above with :+1: so we can see what is popular when we triage.
--------

@Tylerlhess Thank you for opening this issue. 🙏
Please check out these other resources that might help you get to a resolution in the meantime:

- [Check the issue tracker](https://developers.google.com/maps/support/#issue_tracker) - bugs and feature requests for Google Maps Platform APIs and SDKs
- [Open a support case](https://developers.google.com/maps/support#contact-maps-support) - Get 1:1 support in Cloud Console.
- [Discord](https://discord.gg/fHTDmZe9En) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag


_This is an automated message, feel free to ignore._

**Comment by theRocket** [comment]
This is a very good request. For reference, the API documentation now says you must choose which Places API when beginning a project:

https://developers.google.com/maps/documentation/places/web-service/choose-api

> With the release of Places API (New), your first task is to decide which set of APIs to use. This is true if you are a new customer or an existing customer already using the APIs. Use this guide to understand the key differences between the two APIs.

I am mainly interested in the expanded place types, specifically `electric_vehicle_charging_station` as I want to write some optimization logic using python (pandas) dataframes using popular machine learning libraries (such as Tensorflow) and do not want to have to transfer the location data from a Javascript API framework over to a Python framework via file exports since it will be quite large.

**Comment by wangela** [comment]
If you are looking for Go client libraries for the following APIs, see the [Google Maps Platform APIs in the Cloud Client Libraries for Python](https://github.com/googleapis/google-cloud-python/tree/main/packages) ([releases](https://github.com/googleapis/google-cloud-python/releases?q=maps&expanded=true)).

- [Address Validation API](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-addressvalidation)
- [Datasets API](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-mapsplatformdatasets)
- Fleet Engine](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-fleetengine)
- [Places API (New)](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-places)
- [Routes API](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-routing)

The new APIs will not be added to this client library.

**Comment by theRocket** [comment]
@wangela:
> * [Places API (New)](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-places)
> 
> The new APIs will not be added to this client library.

Thanks! Does this library correspond to this documentation?
https://googleapis.dev/python/places/latest/

**Comment by wangela** [comment]
@theRocket Yes, as linked from the [README](https://github.com/googleapis/google-cloud-python/tree/main/packages/google-maps-places#readme). I've noted that the README and documentation text are confusing since they do not refer to the "new" term and the refdocs even call it "v1".

**Comment by Tylerlhess** [comment]
Thanks!

**Comment by karux** [comment]
"@wangela: The new APIs will not be added to this client library."   Why not support the New API?    what alternative python library would you recommend using to obtain a wrapper around the Places (New) API?

**Comment by chemeng** [comment]
Should we expect this library to stop being maintained and transition over to the google cloud client libraries?
